src/_cffi_src/build_openssl.py

```diff
@@ -0,0 +1,80 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
+
+import sys
+
+from _cffi_src.utils import (
+    build_ffi_for_binding
+)
+
+from cryptography.hazmat.bindings.utils import _get_openssl_libraries
```

Comment:
The only place this function is called is from within this build script. This is also the only place that ``_cffi_src`` imports ``cryptography``. I think we should just define this function inside of this file and remove any dependency that ``_cffi_src`` has on ``cryptography``.